client/src/components/ProfilePage/Location/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Inputs from '../../common/Inputs';
+import Button from '../../common/Button';
+import './style.css';
+
+class Location extends Component {
+  state = {
+    validation: false,
+    found: false,
+  };
+
+  onChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value, validation: false, found: false });
+  };
+
+  onClick=() => {
+    const { City, State } = this.state;
+    if (City && City.trim() && State && State.trim()) {
+      const apiKey = '219fa7fc98c345d1b38193fa9201a96e';
+      axios.get(`https://api.opencagedata.com/geocode/v1/json?q=${State},${City}&key=${apiKey}`).then((res) => {
+        if (res.data.total_results !== 0) {
+          const locationValues = res.data.results[0].geometry;
+          const { lat, lng } = locationValues;
+          const data = { State, lat, lng };
+          axios.post('api/v1/geo-code', data).then(
+            this.setState({ message: 'Successfully added ', validation: false, found: false }),
+          ).catch((error) => {
+            const { status } = error.response;
+            if (status === 404) {
+              this.setState({ message: 'Page Not Found' });
+            }
+          });
+        } else {
+          this.setState({ messageNotFound: ' Not Found', found: true });
+        }
+      }).catch((error) => {
+        const { status } = error.response;
+        if (status === 404) {
+          this.setState({ message: ' Not Found' });
```

Comment:
@ali-7 I can't figure out the difference between `message` and `messageNotFound`, so a comment might help (or maybe more descriptive variable names).
I thought `message` was for displaying success messages and `messageNotFound` was for displaying errors, but this line (41) looks like an error message being put into `message` so I must be wrong.

Reply:
@tomduggan85 i put comments in the code to clear it out.
thanks tom :)